pyproject.toml

```diff
@@ -100,6 +100,7 @@ PyYAML = "~5.4.1"
 social-auth-app-django = "^4.0.0"
 # Rendering of SVG images (for rack elevations, etc.)
 svgwrite = "~1.4.1"
+graphene-django-optimizer = "^0.8.0"
```

Comment:
Please alpha sort this requirement and add the comment above it. Also change the version to `"~0.8.0"` (with a tilde), and bring it to whatever the latest version is at the current time.